library/core/src/option.rs

```diff
@@ -980,6 +979,21 @@ impl<T> Option<T> {
         }
     }
 
+    /// Ferrocene addition: `msg` is type `&'static str` instead of `&str`.
```

Comment:
it's actually possible to put `cfg`s directly on function arguments: https://play.rust-lang.org/?version=stable&mode=debug&edition=2024&gist=22369350a86e04ed53b5b8fdc3dcfb9c

could you do that here please?